### DISCUSSION TITLE
Process for upgrade v12.2 to v13.1 on Synology

### DISCUSSION THREAD

**Discussion by hornickj** [alexjustesen/speedtest-tracker#896] [Q&A]
I haven't been able to locate the process to update to the most recent versions, is there a guide or way to accomplish this task without having to rip and replace?  I created this originally within Portainer.

**Comment by alexjustesen** [comment]
So not sure about Portainer as I used their Container Manager instead. I always specify the version in my `yml` file so when I change it CM will automatically rebuild the project with that version

**Comment by hornickj** [comment]
Can you provide an example of that process?

Joey

(sent from my mobile device, please excuse typos)

On Nov 11, 2023, at 4:20 PM, Alex Justesen ***@***.***> wrote:

﻿
This message was sent from outside the company. Please do not click links or open attachments unless you recognize the source of this email and know the content is safe.

So not sure about Portainer as I used their Container Manager instead. I always specify the version in my yml file so when I change it CM will automatically rebuild the project with that version

**Reply by alexjustesen** [reply]
[Container Manager](https://www.synology.com/en-us/dsm/feature/container-manager) just uses the Docker Compose file found in the [docs](https://docs.speedtest-tracker.dev/getting-started/installation/installation) to create projects.
